src/index.js

```diff
@@ -2,7 +2,11 @@
 import Adapter from './Adapter';
 import Gun from 'gun/gun';
 
-Gun.on('opt', (context) => {
+Gun.on('opt', function(context) {
```

Comment:
The "Psycho" in "PsychoLlama" stands for my usage of spaces before a function paren. Changing that should fix the build